tests/e2e/tests/basic.ts

```diff
@@ -0,0 +1,62 @@
+import { Selector } from 'testcafe';
+import { readFileSync } from 'fs';
+import * as path from 'path';
+import * as got from 'got';
+
+
+const maxAppWarmup = 15;
+const rootURL = process.env.SIGNHASH_URL || 'http://localhost:8080';
+
+
+const knownFile = {
+  path: 'foo.txt',
+  name: 'foo.txt',
+  hash: readFileSync(path.join(__dirname, 'foo.hash')).toString().trim()
+};
+
+
+fixture`SignHash`
+  .page(rootURL)
+  .before(waitForPage(maxAppWarmup));
+
+
+test('Site is available', async t => {
+  await t
+    .expect(Selector('body').textContent)
+    .contains('SignHash');
+});
+
+
+test('Verifying known hash', async t => {
+  const file = knownFile;
+  const text = Selector('body').textContent
+
+  await t
+    .setFilesToUpload('#file-upload', file.path)
+    .expect(text).contains(file.name)
+    .expect(text).contains(file.hash)
+    ;
```

Comment:
I'd rather have a semicolon at the end of the previous line. This is a matter of preferred style, and I know that it makes `git diff` better, but it is unusual.

Reply:
Yeah, it's unusual. I don't have any strong opinion on that, it just felt more suitable to this code. It's not anything important at this point so, whatever works.